### ISSUE TITLE
Embedded templates incorrectly violate `no-trailing-spaces` rule

### ISSUE THREAD

**Issue by robinborst95** [ember-template-lint/ember-template-lint#2832]
Embedded templates with the ending backtick on a newline cause a violation of the `no-trailing-spaces` rule, which feels incorrect. Example:
```js
test('it renders', async (assert) => {
  await render(hbs`
    <div class="parent">
      <div class="child"></div>
    </div>
  `);
);
```
